common/src/models/domain/globalState.ts

```diff
@@ -0,0 +1,24 @@
+export class GlobalState {
+    enableAlwaysOnTop?: boolean;
+    installedVersion?: string;
+    lastActive?: number;
```

Comment:
Is this used to determine timeout? Wondering if we should count inactivity per account.

Reply:
We do, and account's last activity is what is used for vault timeout. We track activity even when logged out, though, and while I'm not sure we _need_ to do that, I needed a place to store that information when there aren't any account.

Reply:
Fair enough.